entities/entities/spawned_money/init.lua

```diff
@@ -30,7 +30,7 @@ end
 
 function ENT:Touch(ent)
 	-- the .USED var is also used in other mods for the same purpose
-	if ent:GetClass() ~= "spawned_money" or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
+	if not ent.IsSpawnedMoney or self.USED or ent.USED or self.hasMerged or ent.hasMerged then return end
```

Comment:
Now, we're talking about that function anyway. Change the function to [ENT:StartTouch](http://wiki.garrysmod.com/page/ENTITY/StartTouch), so it's not called continuously?